sentinel-core/src/main/java/com/alibaba/csp/sentinel/traffic/rule/traffic/router/match/MethodMatch.java

```diff
@@ -0,0 +1,197 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.alibaba.csp.sentinel.traffic.rule.traffic.router.match;
+
+import java.util.List;
+import java.util.Map;
+
+import com.alibaba.csp.sentinel.traffic.TrafficContext;
+import com.alibaba.csp.sentinel.util.StringUtil;
+
+/**
+ * @author panxiaojun233
+ */
+public class MethodMatch {
+
+	private StringMatch name;
+	private StringMatch path;
+	private Integer argc;
+	private List<ArgumentMatch> args;
+	private List<StringMatch> argp;
```

Comment:
This seems perplexing... Maybe we could improve the naming.

Reply:
done